src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license.
 
 #include "pch.h"
+#include <WinUser.h>

```

Comment:
this might belong in the `pch.h` in this project with the other platform includes. I'm very surprised it isn't already included!